src/.nuget/descriptions.json

```diff
@@ -8,5 +8,10 @@
         "Name": "Microsoft.NETCore.Runtime.CoreCLR",
         "Description": "The .NET Core runtime, called CoreCLR, and the base library, called mscorlib. It includes the garbage collector, JIT compiler, base .NET data types and many low-level classes.",
         "CommonTypes": [ ]
+    },
+    {
+        "Name": "Microsoft.NETCore.Runtime.CoreCLR.symbols",
+        "Description": "The .NET Core runtime symbol files",
+        "CommonTypes": [ ]
```

Comment:
Is this used? (I'm imagining it's left over from the previous approach, but maybe I'm wrong.)